CHANGELOG.md

```diff
@@ -20,19 +24,18 @@ This parameter enables user to decide whether or not filter should be validated.
 - Added `get_survey_list` into `Mindful` Source file.
 
 
-### Changed
-- Updated `genesys_to_adls.py` flow with the `adls_bulk_upload` task
-- Updated `mindful_to_adls.py` flow with the `adls_bulk_upload` task
-- Changed `MindfulToCSV` task to download surveys info.
-
-
 ## [0.4.11] - 2022-12-15
 ### Added
 - Added into `Genesys` the new view type `AGENT`. 
 
 ### Changed
 - Changed data extraction logic for `Outlook` data.
 
+### Added 
```

Comment:
I think you can delete this part because you already added this under `unreleased`